mq/plugins/ipFixup.py

```diff
@@ -48,12 +48,10 @@ def onMessage(self, message, metadata):
         Also sets ipv4 in two fields:
             ipaddress (decimal mapping IP)
             ipv4address (string mapping)
-            Elastic search is inconsistent about returning IPs as
-            decimal or IPs.
-            In a query an IP field is returned as string.
-            In a facets an IP field is returned as decimal.
```

Comment:
facets was removed in later versions of ES after 1.0
aggregations replaced it, and this field now handles ipv4 and ipv6, though more testing is needed in ES 6 to determine if it behaves accurately. Only modified the wording here to reflect accurately.